coordinator/src/models/errors/validation-errors.ts

```diff
@@ -24,6 +24,25 @@ export class MedicationRequestValueError<T> implements ValidationError {
   }
 }
 
+export class MedicationRequestNumberError implements ValidationError {
+  operationOutcomeCode = "value" as const
+  severity = "error" as const
+  message = `Expected exactly one MedicationRequest in a prescriptionOrderUpdate message`
+  expression = [`Bundle.entry.resource.ofType(MedicationRequest)`]
+}
+
+export class MedicationRequestMissingValueError implements ValidationError {
+  message: string
```

Comment:
kinda dumb that you have to 'define' these again even tho they're already defined in `ValidationError` but guess thats just TS